plugins/utils/todo-comments.nix

```diff
@@ -1,290 +1,429 @@
 {
   lib,
-  helpers,
   config,
   pkgs,
   ...
 }:
 with lib;
 let
-  cfg = config.plugins.todo-comments;
-
-  commands = {
+  inherit (lib.nixvim)
+    defaultNullOpts
+    keymaps
+    mkPackageOption
+    mkCompositeOption
+    ;
+  types = lib.nixvim.nixvimTypes;
+
+  keymapsActions = {
     todoQuickFix = "TodoQuickFix";
     todoLocList = "TodoLocList";
     todoTrouble = "TodoTrouble";
     todoTelescope = "TodoTelescope";
   };
```

Comment:
Up to you, we could drop this, since we have `lib.nixvim.upperFirstChar`: https://github.com/nix-community/nixvim/blob/78fc4be6a830e8dc01f3e66ddbe3243b4bfe8560/lib/utils.nix#L81-L104

Reply:
Well, I guess it's used in the option declaration as well as the keymap transformation. So maybe best to keep it?